minidump/src/minidump.rs

```diff
@@ -5654,25 +5460,22 @@ mod test {
 
     #[test]
     fn test_linux_maps() {
-        // Whitespace intentionally wonky to test robustness
-        let input = b"
+        use procfs::process::{MMPermissions, MMapPath};
 
- a90206ca83eb2852-b90206ca83eb3852 r-xp  10bac9000 fd:05 1196511 /usr/lib64/libtdb1.so\x20\x20
-c70206ca83eb2852-de0206ca83eb2852  -w-s  10bac9000 fd:05 1196511 /usr/lib64/libtdb2.so  (deleted)
-
-
-";
+        // TODO: is it okay to give up wonky whitespace support?
```

Comment:
We can live without it given that the kernel output is unlikely to change dramatically over time

Reply:
I was wondering whether this support existed because of real examples seen in the wild. But I didn't really think that likely given it's probably been static for quite a while in the kernel. At which point I was fearful that we need to support _very_ old systems. Though obviously it's not a difficult thing to add to `procfs`.